packages/components-date/src/FieldDate/FieldDate.stories.tsx

```diff
@@ -106,6 +125,24 @@ Controlled.parameters = {
   storyshots: { disable: true },
 }
 
+export const ControlledFloatingLabel = () => {
+  const [today, setToday] = useState<Date>()
+  const onClickSelectToday = () => setToday(new Date())
+  return (
+    <ExtendComponentsThemeProvider
+      themeCustomizations={{ defaults: { externalLabel: false } }}
+    >
+      <Button onClick={onClickSelectToday}>Today</Button>
+      <FieldDate
+        externalLabel={false}
+        label="Controlled"
+        value={today}
+        onChange={setToday}
+      />
+    </ExtendComponentsThemeProvider>
+  )
+}
+
```

Comment:
Can you disable storyshots on this? I just realized that the snapshot will fail each time the month changes because without a starting value, the calendar defaults to "this month"
```suggestion
ControlledFloatingLabel.parameters = {
  storyshots: { disable: true },
}
```

Reply:
good point!

Reply:
I also removed the snapshot for `FloatingLabelNoDefaultValue`, because this one shouldn't have a default value every time its renders the calendar displayed will refer to the month we are in.